Deepesh/feature1_first_file .txt

```diff
@@ -0,0 +1 @@
+Adding first file
```

Comment:
improve code with more description

Reply:
description is added and updated in the PR.